src/main/java/org/cascadebot/cascadebot/data/objects/Tag.java

```diff
@@ -0,0 +1,99 @@
+/*
+  * Copyright (c) 2019 CascadeBot. All rights reserved.
+  * Licensed under the MIT license.
+ */
+
+package org.cascadebot.cascadebot.data.objects;
+
+import org.apache.commons.lang.math.NumberUtils;
+import org.cascadebot.cascadebot.commandmeta.CommandContext;
+import org.cascadebot.cascadebot.utils.FormatUtils;
+
+import java.time.OffsetDateTime;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.BiFunction;
+import java.util.function.Function;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Tag {
+
+    private static final Map<String, BiFunction<CommandContext, String[], String>> placeholders = new HashMap<>();
+    // https://regex101.com/r/hlsgVW/1
+    private static final Pattern TAG_PATTERN = Pattern.compile("\\{([A-z]+)(?::((?:,?\\w+)+))?}");
+
+    private String content;
+    private String category;
+
+    static {
+        // Guild
+        addPlaceholder("server_region", (context -> context.getGuild().getRegion().getName()));
+        addPlaceholder("server_name", (context -> context.getGuild().getName()));
+        addPlaceholder("server_owner", (context -> context.getGuild().getOwner().getEffectiveName()));
+        addPlaceholder("member_count", (context -> String.valueOf(context.getGuild().getMembers().size())));
+
+        // Sender
+        addPlaceholder("sender", (context -> context.getMember().getEffectiveName()));
+        addPlaceholder("sender_id", (context -> context.getUser().getId()));
+        addPlaceholder("sender_mention", (context -> context.getMember().getAsMention()));
+        
```

Comment:
we should have one of these for getting a different user.

Reply:
How would that work, by id or name? I'm interested to see your use case for that

Reply:
We could hook into the utils we allready have for it. although thinking about it this might be better suited for the full custom command system so the user can better handle it when a user can't be found.

Reply:
Yeah, I don't really see why you'd necessarily want to tag another user. We could add a {user:id} tag that would get a user by static id

Reply:
The main reason is having a tag with info or like a solution to a common issue on like a support server and you want to ping the user having issues. although like i said this would be more suited to the full blown custom commands.